### PR TITLE
Updating Datetime Transformers

### LINKED ISSUE #368: Update `DateTimeTransformer` to make it user friendly
### Problem Description

Let's update `DateTimeTransformer` to make it more user friendly.

### Expected behavior

Name: `UnixTimestampConverter`
Functionality:
- Missing value handling (#362 )
- Remove `strip_constant`
- Reverse to the correctly formatted string
- Change `columns` to `column`

```python
>>> from transformers.datetime import UnixTimestampConverter

# never strips the constant
>>> utc = UnixTimestampConverter()

# format specifies how the data is written (eg. in this case 'Jan 01 2021')
>>> utc = UnixTimestampConverter(format='%b %d %Y')
>>> utc.fit(data, column='datetime_column')
>>> transformed = utc.transform(data)
# reverse back into the correctly formatted string
>>> reversed_data = utc.reverse_transform(transformed)
```

### PR DESCRIPTION
**Pull request by amontanez24**
resolves #368 and #369

### CODE REVIEW

File: rdt/transformers/datetime.py [hunk 1]
```diff
@@ -25,13 +25,7 @@ class DatetimeTransformer(BaseTransformer):
...
             there are null values. If ``False``, do not create the new column even if there
             are null values. Defaults to ``False``.
-        strip_constant (bool):
-            Whether to optimize the output values by finding the smallest time unit that
-            is not zero on the training datetimes and dividing the generated numerical
-            values by the value of the next smallest time unit. This, a part from reducing the
-            orders of magnitued of the transformed values, ensures that reverted values always
-            are zero on the lower time units.
-        format (str):
+        datetime_format (str):
```
Comment (pvk-developer) [inline rdt/transformers/datetime.py:28]: In the issue says `format` do we want to change it to `datetime_format` ?

**Review by pvk-developer — APPROVED** [review]
Overall looks good, just minor (maybe I'm out of context) the issue says `format` and not `daytime_format` , which one should we keep ?

**Review by fealho — APPROVED** [review]
Looks good 💯

File: rdt/transformers/datetime.py (hunk 1 above)
Comment (amontanez24) [reply rdt/transformers/datetime.py:28]: I don't think I can make it `format` because `format` is a special python function

**Pull request merged**